OrbitCore/Path.cpp

```diff
@@ -321,36 +322,27 @@ void Path::Dump() {
 bool Path::IsSourceFile(const std::string& a_File) {
   std::string ext = Path::GetExtension(a_File);
   return ext == ".c" || ext == ".cpp" || ext == ".h" || ext == ".hpp" ||
-         ext == ".inl" || ext == ".cxx";
+         ext == ".inl" || ext == ".cxx" || ext == ".cc";
 }
 
 std::vector<std::string> Path::ListFiles(
```

Comment:
can you add a test? you can use a directory in testdata for this.

Reply:
This has nothing to do with the current change actually, just stumbled upon that method and realized it was missing (at least) one extension.   A test is a good idea, but for a subsequent PR.